src/Nncase.Passes/Rules/Neutral/FoldLayerNorm.cs

```diff
@@ -333,3 +338,63 @@ public sealed partial class FoldLayerNormPattern5 : RewriteRule<CallPattern>
         return null;
     }
 }
+
+[RuleGenerator]
+public sealed partial class ConvertLayerNormChannelFirstToLast : RewriteRule<CallPattern>
+{
+    public override CallPattern Pattern { get; } =
+        IsLayerNorm(
+            "ln",
+            "_",
+            _ => true,
+            IsWildcard("x"),
+            IsWildcard("scale"),
+            IsWildcard("bias"));
+
+    private static List<int> GetPermWithAxis(long axis, int shapeSize)
+    {
```

Comment:
our layernorm definition comes from ONNX，**axis** mean the start dim to do layernorm([LN](https://github.com/onnx/onnx/blob/main/docs/Operators.md#layernormalization)), so just using axis may not be very proper to tell channel last from onnx's layernorm.

Reply:
Added 'Arg: ChannelFirst' to distinguish it from the ONNX LayerNorm implementation